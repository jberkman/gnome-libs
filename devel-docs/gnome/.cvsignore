Makefile
Makefile.in
html
sgml
gnome.html
