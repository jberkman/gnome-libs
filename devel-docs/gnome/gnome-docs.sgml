<!doctype book PUBLIC "-//Davenport//DTD DocBook V3.0//EN" [
<!entity gnome-gnome-config SYSTEM "sgml/gnome-config.sgml">
<!entity gnome-gnome-defs SYSTEM "sgml/gnome-defs.sgml">
<!entity gnome-gnome-dentry SYSTEM "sgml/gnome-dentry.sgml">
<!entity gnome-gnome-exec SYSTEM "sgml/gnome-exec.sgml">
<!entity gnome-gnome-fileconvert SYSTEM "sgml/gnome-fileconvert.sgml">
<!entity gnome-gnome-help SYSTEM "sgml/gnome-help.sgml">
<!entity gnome-gnome-history SYSTEM "sgml/gnome-history.sgml">
<!entity gnome-gnome-i18n SYSTEM "sgml/gnome-i18n.sgml">
<!entity gnome-gnome-metadata SYSTEM "sgml/gnome-metadata.sgml">
<!entity gnome-gnome-mime-info SYSTEM "sgml/gnome-mime-info.sgml">
<!entity gnome-gnome-mime SYSTEM "sgml/gnome-mime.sgml">
<!entity gnome-gnome-paper SYSTEM "sgml/gnome-paper.sgml">
<!entity gnome-gnome-popt SYSTEM "sgml/gnome-popt.sgml">
<!entity gnome-gnome-regex SYSTEM "sgml/gnome-regex.sgml">
<!entity gnome-gnome-remote SYSTEM "sgml/gnome-remote.sgml">
<!entity gnome-gnome-score SYSTEM "sgml/gnome-score.sgml">
<!entity gnome-gnome-sound SYSTEM "sgml/gnome-sound.sgml">
<!entity gnome-gnome-triggers SYSTEM "sgml/gnome-triggers.sgml">
<!entity gnome-gnome-url SYSTEM "sgml/gnome-url.sgml">
<!entity gnome-gnome-util SYSTEM "sgml/gnome-util.sgml">
<!entity gnome-libgnome SYSTEM "sgml/libgnome.sgml">
]>

<book>
  <bookinfo>
    <title>Gnome Library Reference Manual</title>
  </bookinfo>

  <chapter id="libgnome">
    <title>Gnome Library</title>
    &gnome-gnome-config;
    &gnome-gnome-defs;
    &gnome-gnome-dentry;
    &gnome-gnome-exec;
    &gnome-gnome-fileconvert;
    &gnome-gnome-help;
    &gnome-gnome-history;
    &gnome-gnome-i18n;
    &gnome-gnome-metadata;
    &gnome-gnome-mime-info;
    &gnome-gnome-mime;
    &gnome-gnome-paper;
    &gnome-gnome-popt;
    &gnome-gnome-regex;
    &gnome-gnome-remote;
    &gnome-gnome-score;
    &gnome-gnome-sound;
    &gnome-gnome-triggers;
    &gnome-gnome-url;
    &gnome-gnome-util;
    &gnome-libgnome;
  </chapter>
</book>
