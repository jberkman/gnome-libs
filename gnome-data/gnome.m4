AC_DEFUN([GNOME_CHECK_GNOME],
[
	AC_REQUIRE([GNOME_X_CHECKS])
	AC_CHECK_LIB(gnome, gnome_init, [GNOME_LIBS="-lgnome -lgnomeui"],[
		AC_MSG_ERROR(Gnome libraries not found)],[
		"$GTK_LIBS $x_libs"])	
])

AC_DEFUN([GNOME_INIT],
[	
	AC_SUBST(GNOME_LIBS)
	AC_SUBST(GNOMEUI_LIBS)
	AC_SUBST(GTKXMHTML_LIBS)
	AC_SUBST(GNOMELIB_DIR)
	AC_SUBST(GNOMEINCLUDE_DIR)

	if test x$prefix = xNONE; then
	    gnome_prefix=$ac_default_prefix/lib
	else
	    gnome_prefix=$prefix/lib
	fi

	AC_ARG_WITH(gnome-includes,
	[--with-gnome-includes	Specify location of GNOME headers],[
	CFLAGS="$CFLAGS -I$withval"
	])

	AC_ARG_WITH(gnome-libs,
	[--with-gnome-libs	Specify location of GNOME libs],[
	LDFLAGS="$LDFLAGS -L$withval"
	gnome_prefix=$withval
	])

	AC_ARG_WITH(gnome,
	[--with-gnome		Specify prefix for GNOME files],[
	LDFLAGS="$LDFLAGS -L$withval/lib"
	CFLAGS="$CFLAGS -I$withval/include"
	gnome_prefix=$withval/lib
	])
	

        AC_MSG_CHECKING(for gnomeConf.sh file)
	if test -e $gnome_prefix/gnomeConf.sh; then
	    AC_MSG_RESULT(found)
	    echo "loading gnome configuration from $gnome_prefix/gnomeConf.sh"
	    . $gnome_prefix/gnomeConf.sh
	else
	    AC_MSG_ERROR(Could not find the gnomeConf.sh file that is generated by gnome-libs install)
	fi
])
