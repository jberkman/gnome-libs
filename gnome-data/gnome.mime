#
# (C) The Free Software Foundation, 1998
#
# Mime types as provided by the GNOME libraries for GNOME.
#
# Applications can provide more mime types by installing other
# .mime files in the $share/mime-info directory.
#
# The format of this file is:
#
# mime-type
#         ext[,prio]:   list of extensions for this mime-type
#         regex[,prio]: a regular expression that matches the filename
# 
# more than one ext: and regex: fields can be present.
# 
# prio is the priority for the match, the default is 1.  This is required 
# to distinguish composed filenames, for example .gz has a priority of 1
# and .tar.gz has a priority of 2 (thus a file having the filename
# something.tar.gz will match the mime-type for tar.gz before the mime-type
# for .gz
#

application/x-gnome-app-info 	
	ext: desktop

text/c				
	ext: c h

text/C++			
	ext: cc C cpp

text/x-perl
	ext: pl perl

text/x-scheme
	ext: scm

text/x-python:
	ext: py

application/x-bzip		
	ext: bz2 bz

audio/x-wav:
	ext: wav

audio/x-mp3			
	ext: mp3

audio/x-aiff			
	ext: aif aifc aiff

audio/ulaw			
	ext: au

audio/basic			
	ext: snd

audio/x-voc			
	ext: voc

application/vcard		
	ext: vcf

application/x-gnome-card
	ext: gcrd

application/x-troff-man		
	regex: (([^0-9]|^[^\.]*)\.([1-9][A-Za-z]*|n)|\.man)$

application/x-troff-man-compressed
	regex: ([^0-9]|^[^\.]*)\.([1-9][a-z]?|n)\.g?[Zz]$

x-url/http
	regex: http://.*

x-url/ftp
	regex: ftp://.*

x-url/info
	regex: info:.*

x-url/gopher
	regex: gopher:.*

x-url/news
	regex: news:.*

image/x-bmp			
	ext: bmp

image/x-png			
	ext: png

image/x-tga			
	ext: tga

application/x-object-file	
	ext: o

application/x-shared-library
	ext: so

application/postscript
	ext: ps eps

application/pgp
	ext: pgp
audio/x-aiff
	ext: aif aifc aiff

text/plain
	ext: asc txt

audio/ulaw
	ext: au

application/x-bcpio
	ext: bcpio

application/octet-stream
	ext: bin

application/x-netcdf
	ext: cdf nc

application/x-cpio
	ext: cpio

application/x-cpio-compressed
	regex: \.cpio\.gz$

application/x-csh
	ext: csh

application/x-dvi
	ext: dvi

text/x-setext
	ext: etx

application/andrew-inset
	ext: ez

image/gif
	ext: gif

application/x-gtar
	ext: gtar

application/x-gunzip
	ext: gz

application/x-hdf
	ext: hdf

text/html
	ext: html

image/ief
	ext: ief

image/jpeg
	ext: jpe jpeg jpg

application/x-latex
	ext: latex

application/x-troff-man
	ext: man

application/x-troff-me
	ext: me

application/x-mif
	ext: mif

video/quicktime
	ext: mov qt

video/x-sgi-movie
	ext: movie

video/x-msvideo:
	ext: avi

video/mpeg
	ext: mp2 mpe mpeg mpg

video/x-msvideo
	ext: avi

application/x-troff-ms
	ext: ms

application/oda
	ext: oda

image/x-portable-bitmap
	ext: pbm

application/pdf
	ext: pdf

image/x-portable-graymap
	ext: pgm

application/x-chess-pgn
	ext: pgn

image/x-portable-anymap
	ext: pnm

image/x-portable-pixmap
	ext: ppm

image/x-cmu-raster
	ext: ras

image/x-rgb
	ext: rgb

application/x-troff
	ext: roff t tr

application/rtf
	ext: rtf

text/richtext
	ext: rtx

application/x-sh
	ext: sh

application/x-shar
	ext: shar

audio/basic
	ext: snd

application/x-wais-source
	ext: src

application/x-sv4cpio
	ext: sv4cpio

application/x-sv4crc
	ext: sv4crc

application/x-tar
	ext: tar

application/x-compressed-tar
	regex,2: tar\.gz$
	ext: tgz

application/x-tcl
	ext: tcl

application/x-tex
	ext: tex

application/x-texinfo
	ext: texi texinfo

image/tiff
	ext: tif tiff

text/tab-separated-values
	ext: tsv

application/x-ustar
	ext: ustar

audio/x-wav
	ext: wav

image/x-xbitmap
	ext: xbm

image/x-xpixmap
	ext: xpm

image/x-xwindowdump
	ext: xwd

application/zip
	ext: zip

audio/x-pn-realaudio      
	ext: rm ra ram

application/x-rpm
	ext: rpm

application/x-deb
	ext: deb

application/x-zoo
	ext: zoo

application/x-lha
	ext: lha

application/x-arj
	ext: arj

application/x-unix-archive
	ext: a sa

text/fortran:
	ext: f

application/x-gnu-rmail
	regex: ^RMAIL$
