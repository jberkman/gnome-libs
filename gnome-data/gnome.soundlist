[__section_info__]
description=GNOME system events
description[ca]=Events de sistema de GNOME
description[da]=Systemhændelser i Gnome skrivebordsmiljøet
description[de]=Sound-Ereignisse für GNOME
description[el]=Ãåãïíüôá ôïõ óõóôŞìáôïò ôïõ GNOME
description[es]=Eventos sistema de GNOME
description[et]=GNOME süsteemsed sündmused
description[fr]=Événements système de GNOME
description[gl]=Eventos de sistema de GNOME
description[it]=Eventi di sistema GNOME
description[ko]=±×³ğ ½Ã½ºÅÛ ÀÌº¥Æ®
description[lt]=GNOME sistemos ávykiai
description[no]=Systemhendelser for GNOME
description[pl]=Zdarzenia systemowe GNOME
description[pt]=Eventos de sistema GNOME
description[tr]=Gnome sistem eylemleri
description[wa]=Evenmints sistinme di GNOME

[login]
file=login.wav
description=Login
description[ca]=Entrada al sistema
description[da]=Indlogning
description[de]=Anmeldung zur Sitzung
description[el]=¸íáñîç óõíåäñßáò
description[es]=Apertura de sesión
description[et]=Sisselogimine
description[fr]=Ouverture de session
description[gl]=Comezo de sesión
description[it]=Login
description[ko]=·Î±×ÀÎ
description[lt]=Prisijungimas
description[no]=Innlogging
description[pl]=Rozpoczêcie pracy
description[pt]=Início de Sessão
description[tr]=Giriş
description[wa]=Intreye è session

[logout]
file=logout.wav
description=Logout
description[ca]=Eixida del sistema
description[da]=Udlogning
description[de]=Abmeldung von der Sitzung
description[el]=¸îïäïò áğü ôç óõíåäñßá
description[es]=Salida de sesión
description[et]=Väljalogimine
description[fr]=Fin de session
description[gl]=Fin de sesión
description[it]=Esci da GNOME
description[ko]=·Î±×¾Æ¿ô
description[lt]=Atsijungimas
description[no]=Utlogging
description[pl]=Zakoñczenie pracy
description[pt]=Fim de Sessão
description[tr]=Çıkış
description[wa]=Cwand li session est hûte

[info]
file=info.wav
description=Informational Messages
description[ca]=Missatges informatius
description[da]=Informative beskeder
description[de]=Informationen
description[el]=ĞëçñïöïñéáêÜ ìçíıìáôá
description[es]=Mensajes informativos
description[et]=Teated
description[fr]=Messages d'information
description[gl]=Mensaxes de información
description[it]=Informazioni
description[ko]=¾Ë¸² ¸Ş¼¼Áö
description[lt]=Informacinës şinutës
description[no]=Informative meldinger
description[pl]=Komunikat informacyjny
description[pt]=Mensagens Informativas
description[tr]=Bilgi iletileri
description[wa]=Messaedjes d' informåcion

[warning]
file=warning.wav
description=Warning Messages
description[ca]=Missatges d'avís
description[da]=Advarsler
description[de]=Warnungen
description[el]=Ìçíıìáôá ğñïåéäïğïßçóçò
description[es]=Mensajes de advertencia
description[et]=Hoiatused
description[fr]=Messages d'avertissement
description[gl]=Mensaxes de aviso
description[it]=Avviso
description[ko]=°æ°í ¸Ş¼¼Áö
description[lt]=Áspëjamosios şinutës
description[no]=Advarsler
description[pl]=Komunikat ostrzegawczy
description[pt]=Avisos
description[tr]=Ikaz iletileri
description[wa]=Messaedjes d' avertixhmint

[error]
file=error.wav
description=Error Messages
description[ca]=Missatges d'error
description[da]=Fejlmeldinger
description[de]=Fehler
description[el]=Ìçíıìáôá óöÜëìáôïò
description[es]=Mensajes de error
description[et]=Veateated
description[fr]=Messages d'erreur
description[gl]=Mensaxes de erro
description[it]=Errore
description[ko]=¿À·ù ¸Ş¼¼Áö
description[lt]=Klaidø praneğimai
description[no]=Feilmeldinger
description[pl]=Komunikat b³êdu
description[pt]=Mensagens de Erro
description[tr]=Hata iletileri
description[wa]=Messaedjes d' aroke

[question]
file=question.wav
description=Question Dialogs
description[ca]=Diàlegs interrogatius
description[da]=Spørgsmålsdialoger
description[de]=Fragen
description[el]=ÄéÜëïãïé ìå åñùôŞóåéò
description[es]=Diálogos con preguntas
description[et]=Küsimused
description[fr]=Dialogues avec des questions
description[gl]=Diálogos con preguntas
description[it]=Domanda
description[ko]=¹°À½ ´ëÈ­»óÀÚ
description[lt]=Klausimø dialogai
description[no]=Spørsmålsdialoger
description[pl]=Komunikat z zapytaniem
description[pt]=Diálogos com questões
description[tr]=Sorgulama iletileri
description[wa]=Purneas di dialogue avou des kesses

[generic]
file=generic.wav
description=Miscellaneous Messages
description[ca]=Missatges diversos
description[da]=Diverse beskeder
description[de]=Sonstige Meldungen
description[el]=ÄéÜöïñá ìçíıìáôá
description[es]=Mensajes misceláneos
description[et]=Muud teated
description[fr]=Autres messages
description[gl]=Outras mensaxes
description[it]=Evento generico
description[ko]=±× ¿Ü ¸Ş¼¼Áö
description[lt]=Ávairios şinutës
description[no]=Forskjellige beskjeder
description[pl]=Ró¿ne komunikaty
description[pt]=Outras mensagens
description[tr]=Öbür iletileri
description[wa]=Ôtes messaedjes
