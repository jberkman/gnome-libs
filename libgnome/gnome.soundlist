[events]
startup=startup.wav
shutdown=shutdown.wav
info=info.wav
warning=warning.wav
error=error.wav
question=question.wav
generic=generic.wav
