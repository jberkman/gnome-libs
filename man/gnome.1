.TH GNOME 1 "GNOME 1.0" 
.SH NAME
GNOME \- The GNU Network Object Model Environment
.SH SYNOPSIS
.PP
.B gnome-session
.SH DESCRIPTION
.PP
The 
.B gnome-session
program launches and initializes the GNOME desktop environment.  This
program is usually executed from your X initialization file.  If
your system is configured to use
.B gdm
(the GNOME display and login manager) you can start your GNOME session
by selecting the Gnome login profile.
.PP
GNOME is a collection of libraries and applications.  A collection of
these form the GNOME Desktop: an easy to use, yet powerful desktop
environment for Unix systems.  You can find up to date information
about GNOME in http://www.gnome.org. 
.PP
From a user's point of view, the GNOME desktop consists of a desktop
metaphor, a file manager and an easy way to launch applications
installed on the system.  Various desktop tools are provided with the
GNOME desktop to take advantage of a computer system.  
.PP
GNOME's desktop metaphor allows the desktop to be used as a place to
temporarily storing files, shortcuts to programs and documents.  Drag
and drop is an important part of the system; we have tried to make the
system as intuitive as possible.
.PP
The session management in GNOME will automatically restore all of the
applications you were running when you log in into the system again.
With session managed applications, the user can turn off or logout
from the system and when he logs in again, he will see the same
desktop he had before.
.PP
GNOME supports themes that allow users to change the skin of an
application: the look of applications in the GNOME desktop can be
configured to look in the way that more pleases the user: it is just a
few mouse-clicks away.  You can choose from a wide range of GTK
themes.  A web site has been devoted to this: http://gtk.themes.org
.PP
.SH ARCHITECTURE
The GNOME architecture addresses a number of problems and missing
features found on Unix systems and it uses a number of components to
achieve this:
.PP
.SS glib
This is the foundation library that provides portability functions, a
collection of reusable abtract types for C programmers and a main loop
abstraction.  For more information see http://www.gtk.org/rdp/glib/book1.html
.SS ORBit
This is the CORBA implementation used in GNOME.  CORBA provides basic
RPC functionality and it is the foundation for the component model and
the compound document and document model systems.  For more
information see http://www.labs.redhat.com/orbit.
.SS GTK+
This is the GUI toolkit used by GNOME.  It works on Unix and Win32
systems and other ports are being worked on to lighter windowing
systems.  You can find more information on http://www.gtk.org/
.SS gtk-engines
The GTK+ toolkit has support for changing the apperance of application
by providing support for themes and theme engines.  See
http://gtk.themes.org for a collection of readily-available themes. 
.SS Imlib
The graphics library used to load, save, manipulate and render images
in GNOME applications.  It includes routines to do fast drawing and
use a limited set of colors from low-end displays.
.SS gnome-libs
These libraries are the core libraries that provide the uniformity of
the applications.  They are divided in five: libgnome (for non-GUI
dependant code), libgnomeui (for GUI dependant code), zvt (the
xterm terminal emulator), gtk-xmhtml (an HTML rendering engine) and
libgnorba that implements the CORBA object activation and registry.
.SS Docbook
GNOME documentation is written in the Docbook SGML DTD.  You can find
more about this at
http://nis-www.lanl.gov/~rosalia/mydocs/docbook-intro.html
.SS GNU gettext
GNOME uses the GNU gettext to allow applications to be localized for
various countries and languages.
.PP
GNOME is window manager independant.  This means that the GNOME
desktop and the GNOME tools will work with any window manager.  Window
manager can optionally provide a number of features that will make the
user's desktop a more pleasant experience.  The GNOME window manager
hint spec is available at: http://www.gnome.org/devel/gnomewm/book1.html
.SH HISTORY
There were two projects that lead to the creation of origins of what
became the GNOME project: the libapp project and the old-GNOME
project.  The former was a project to provide standard
workstation-like services to applications.  The old-GNOME project was
intended to provide a component model for Unix systems.  These were
projects some of us had discussed but never actually implemented.
.PP
Enter KDE,  a project that wanted to make Unix usable as a desktop
machine.  Sadly they chose the proprietary and non-free toolkit Qt
as the foundation for their work.  It was a giant step backwards in
terms of software freedom[1].
.PP
In response, the GNOME project was started later to create a
completely free desktop environment, and various early ideas were
reused.
.PP
Early talks about the creation of GNOME involved some recognized free
software leaders: Erik Troan and Mark Ewing of Red Hat software,
Richard Stallman of the Free Software Foundation, and Peter Mattis and
Spencer Kimball of the GIMP project.  We launched the project after
considering the various alternatives that could be tried.
.PP
The original call for developers included the team of programmers
working on the GNU Image Manipulation Program (GIMP [GIMP]), the Guile
mailing list and the free software mailing lists.  This is important
because the mix of people that were part of the original GNOME team
had a good background on free software issues, graphics and language
design.
.PP
Red Hat created the Advanced Development Laboratories division on
January 1998 (http://www.labs.redhat.com).  RHAD labs was initially
created to help out in the development of the GNOME project.
.PP
We made releases of the GNOME source base since the beginning of the
project.  During the development of GNOME, the group has produced a
number of libraries that are useful to provide integration, and
consistency troughout the system.
.PP
GNOME 1.0 was released after eighteen months of development in March
1999.  Updates and fixes are continously released; At the time of this
writing, the GNOME 1.0 series is at version 1.0.5.
.PP
GNOME 1.0 marks the contract between GNOME developers and the user
base to provide a stable API on top of which new applications can be
developed.  Software developers will be able to take advantage of all
the functions available in the library, and they can be sure that
their applications will continue to work in the future.
.PP
.SH BUGS
.PP
To report bugs in the GNOME system, please use the command 
.B gnome-bug
to send us information about the problem you are experimenting, or go
directly to our bug tracking system on the Web at
http://bugs.gnome.org
.SH AUTHOR
.PP
GNOME has been developed by a large number of programmers, users and
enthusiasts on the Internet.  The 
.B guname
program lists some of the contributors. 
.PP
This manual page has been written by Miguel de Icaza (miguel@gnu.org)
