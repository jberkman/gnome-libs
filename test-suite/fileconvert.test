the_test=fileconvert
. ../../../test-suite/runtest.sh
