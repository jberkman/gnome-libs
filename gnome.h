#ifndef GNOME_H
#define GNOME_H

#include "gnomesupport.h"

#include <gtk/gtk.h>
#include "libgnome/libgnome.h"
#include "libgnomeui/libgnomeui.h"

#endif
