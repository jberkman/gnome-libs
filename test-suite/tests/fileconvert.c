#include <gnome.h>
#include <glib.h>
#include <stdio.h>
#include <unistd.h>

/* Do whatever you need to do... You should put the "expected" output of
   the program in ../expected/testname.out, which will be compared against
   the actual output of the test when it is run.

   A non-zero exit code also indicates failure.
*/

int main(int argc, char *argv[])
{
	gint outfd, readlen;
	gchar abuf[128];

	gnomelib_init(&argc, &argv);
	outfd = gnome_file_convert("fileconvert.in",
				   "text/html",
				   "text/ascii");

	while((readlen = read(outfd, abuf, sizeof(abuf) - 1))
		== sizeof(abuf) - 1) {
		write(1, abuf, readlen);
	}

	printf("\n");
	fflush(stdout); /* Make sure to fflush things after you do
			   any output, so what you see on your terminal
			   is the same as what goes into the output file */
	return 0;
}
