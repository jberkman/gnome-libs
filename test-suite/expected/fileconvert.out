
   This is a [1]test of the file conversion routines.


References

   1. file://localhost/tmp/.27297.html#foo

Exit code 0
